packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.test.tsx

```diff
@@ -4,142 +4,130 @@ import { cleanup } from "@testing-library/react";
 import { CivilDateTime } from "@std-proposal/temporal";
 import { FormatRelativeDateTime } from "./FormatRelativeDateTime";
 
-afterEach(cleanup);
 beforeEach(() => {
-  Date.now = jest.fn(() => 1593115122000);
+  jest.useFakeTimers("modern");
+  jest.setSystemTime(new Date(1593115122000));
+});
+
+afterEach(() => {
+  jest.useRealTimers();
+  cleanup();
```

Comment:
Just wondering here!  It's not obvious right away, but why useFakeTimers before and then useRealTimers after?

Reply:
because instead of mocking what the `Date.now` returns (what it used to be), I mocked the system time so I can reset it back easily once i'm done.

<img width="699" alt="Pasted_Image_2021-12-13__5_12_PM" src="https://user-images.githubusercontent.com/15986172/145909281-f49baaf8-e7b8-44f6-af9e-1cb75d70d539.png">

Reply:
Ah so that's why all the `Date.now()`s are replaced with `Date()`.  Was there a reason for this change having to do with Node? Sorry I'm being nosey 😛

Reply:
In a way. Basically removing redundancy resulted to this.

Reply:
Thanks :)